apps/amo/tests/test_url_prefix.py

```diff
@@ -59,6 +59,10 @@ def test_no_redirect(self):
         response = self.process('/services')
         assert response is None
 
+    def test_api_no_redirect(self):
+        response = self.process('/api/v3/')
```

Comment:
Being a crazy person I'd rather see a URL like `'/api/v3/some/endpoint/'` to make sure it isn't just avoiding the redirect for the root.